src/services/user.js

```diff
@@ -215,18 +195,39 @@ const getLocalInfo = () => {
   let token;
 
   try {
-    ({ token } = storage.get('auth'));
+    token = storage.get('token');
   } catch (err) {
-    throw new Error('Could not get local auth');
+    throw new AuthError('Could not get local auth');
   }
 
   try {
     return jwtDecode(token);
   } catch (err) {
-    throw new Error(`Saved token ${token} could not be parsed`);
+    throw new AuthError(`Saved token "${token}" could not be parsed`);
   }
 };
 
+/**
+ * Get the number of cl&w credits aquired, and number of credits required.
+ * @return {Promise.<CLWCredits>} An Object of their current and requiered number of CL&W events,
+ */
+const getChapelCredits = async () => {
+  const { user_name: username } = getLocalInfo();
+  const termCode = session.getTermCode();
+  const attendedEvents = await getAttendedEvents(username, termCode);
+
+  // Get required number of CL&W credits for the user, defaulting to thirty
+  let required = 30;
```

Comment:
this is okay but the required field is in the account view. At some point we have to move over to that api for required credits cuz 30 will still be inaccurate for a large portion of the student body